src/FurCoNZ.Web/ViewModels/ThirtyDayReminderViewModel.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace FurCoNZ.Web.ViewModels
+{
+    public class ThirtyDayReminderViewModel
```

Comment:
May I suggest renaming to `UnpaidReminderViewModel` 

Also the `v` in `viewModel` is lowercase 😉

Reply:
Renamed it. Not sure what you're referring to with the lowercase `v`?